asciidoc/client-applications.adoc

```diff
@@ -348,377 +324,162 @@ include::{javascript-examples}/examples.test.js[tags=custom-auth]
 ----
 ======
 
-[.include-with-python]
-======
-[source, python]
-----
-include::{python-examples}/custom_auth_example.py[tags=custom-auth]
-----
-======
-====
-
-
-[[driver-configuration]]
-== Configuration
-
-
-[[driver-configuration-encryption]]
-=== Encryption
-
-TLS encryption is enabled for all connections by default.
-This can be disabled through configuration as follows:
-
-[.tabbed-example]
-.Unencrypted
+// [.include-with-python]
+// ======
+// [source, python]
+// ----
+// include::{python-examples}/custom_auth_example.py[tags=custom-auth]
+// ----
+// ======
 ====
-[.include-with-dotnet]
-======
-[source, csharp]
-----
-include::{dotnet-examples}/Examples.cs[tags=config-unencrypted]
-----
-======
-
-[.include-with-go]
-======
-[source, go]
-----
-include::{go-examples}/examples_test.go[tags=config-unencrypted]
-----
-======
-
-[.include-with-java]
-======
-[source, java]
-----
-include::{java-examples}/ConfigUnencryptedExample.java[tags=config-unencrypted]
-----
-======
 
-[.include-with-javascript]
-======
-[source, javascript]
-----
-include::{javascript-examples}/examples.test.js[tags=config-unencrypted]
-----
-======
 
-[.include-with-python]
-======
-[source, python]
-----
-include::{python-examples}/config_unencrypted_example.py[tags=config-unencrypted]
-----
-======
-====
+[[driver-security]]
+== Security
 
-The server can be modified to require encryption for all connections.
-Please see the <<operations-manual#connectors, Operations Manual -> Configure Neo4j connectors>> for more information.
 
-An attempt to connect to a server using an encryption setting not allowed by that server will result in a <<driver-service-unavailable, _Service unavailable_>> status.
+[CAUTION]
+There are significant changes to security settings in Neo4j 4.0. Please read this (and/or consult the migration guide) before upgrading from a previous version.
```

Comment:
I know we've discussed before about the presence of this kind of comparisons in the docs and agreed that it may be necessary to have it _right there_ in the Driver Manual. 
None the less I would like to suggest some future-proofing in case we forget to address it when 4.1 is released... I propose:

There are significant changes to security settings between Neo4j 3.x and Neo4j 4.x. Please consider the information in this section before upgrading from a previous version. The <<migration-guide#migration-guide, Migration Guide>> is also a good source of information about changes in functionality.

Reply:
Agreed and fixed.